copy_input_files.sh

```diff
@@ -11,9 +11,10 @@ FILES_INFO=(
   pre/load/current/index.status:1500000
 )
 
-# Delete old input files
+# zip/delete old input files
 if [ -d ./logs/input ]; then
-    find ./logs/input/ -name "input.20*-*-*_*-*-*" -type d -mtime +7 -exec rm -rf '{}' \;
+    find ./logs/input/ -name "input.20*-*-*_*-*-*" -type d -mtime +3 -exec tar cvzf '{}'.tar.gz '{}' \; -exec rm -r '{}' \;
```

Comment:
Good! Even better it would be if we just use gzip, we do not gain much from doing a tar of a single file and when you `gzip file` it removes the original file for you. Also, a smaller mtime would be acceptable here (just keep last day for instance).

Reply:
tar is picking up [4 files] i(https://github.com/adsabs/ADSImportPipeline/blob/master/config.py#L24).  they all have the same name but are in different subdirectories.  I think this makes tar a better choice.  I'll reduce the the mtime

Reply:
My bad, I misinterpreted the find command. Do we know if the rm gets executed only if the tar succeeds? If so, we can go ahead with this version.

Reply:
yes, docs say find runs -execs in order and stops when one fails